### ISSUE TITLE
Getting OSI info from UDP port

### ISSUE THREAD

**Issue by kancanaraa** [esmini/esmini#500]
Hi,

I want to get the OSI messages with using UDP port in python.
But, to do that currently I have to specify UDP controller to the vehicles.
And I dont want to control vehicle, I just want to get OSI infos.

Could you please suggest a solution?

**Comment by eknabevcc** [comment]
Good point. Current example includes both OSI and driver control over UDP, making it a bit messy. 

I added a cleaner example, which only receives OSI data and prints some of it:
https://github.com/esmini/esmini/blob/dev/EnvironmentSimulator/code-examples/hello_world/osi_groundtruth_from_udp.py

Try to run it from esmini root folder. Instructions in file top comment.

It still makes use of UDP and OSI classes of the common:
https://github.com/esmini/esmini/blob/dev/scripts/udp_driver/udp_osi_common.py

Here is another example which includes (duplicates of) those classes, only depending on OSI3 definitions.
[osi_groundtruth_from_udp_including_udp_receiver.zip](https://github.com/esmini/esmini/files/13508931/osi_groundtruth_from_udp_including_udp_receiver.zip)

Hopefully it works for your use case.
